### PR TITLE
KEYCLOAK-18950 Save Button Enabled For Empty Attributes

### PR DESCRIPTION
**Pull request by mabartos**
JIRA: [KEYCLOAK-18950](https://issues.redhat.com/browse/KEYCLOAK-18950)

I'll execute the console tests and share the results here.

Edit: It also helped to show default values for CIBA Policy attributes in Authentication section.

@mposolda @vmuzikar @Aboullos Could you please take a look at it?

### CODE REVIEW

File: server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java [hunk 1]
```diff
@@ -474,7 +474,8 @@ public static RealmRepresentation toRepresentation(KeycloakSession session, Real
 
         session.clientPolicy().updateRealmRepresentationFromModel(realm, rep);
 
-        rep.setAttributes(stripRealmAttributesIncludedAsFields(realm.getAttributes()));
+        // Append realm attributes to representation
+        rep.getAttributes().putAll(stripRealmAttributesIncludedAsFields(realm.getAttributes()));
```
Comment (mabartos) [inline server-spi-private/src/main/java/org/keycloak/models/utils/ModelToRepresentation.java:478]: Attributes in present realm representation was overridden by `RealmModel` attributes. These attributes should be appended.

**Comment by mabartos** [comment]
@mposolda @vmuzikar Pipeline tests passed. See https://keycloak-jenkins.com/job/universal-test-pipeline-server/712/

**Review by Aboullos — APPROVED** [review]
LGTM!

**Review by mposolda — APPROVED** [review]
@mabartos @Aboullos Thanks!

**Pull request merged**